README.md

```diff
@@ -1,13 +1,237 @@
 # 💪 strong-config
 
-Secret management for NodeJS. Based on [SOPS](https://github.com/mozilla/sops).
+<!-- markdownlint-disable line-length -->
+[![pipeline status](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/pipeline.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master) [![coverage report](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/coverage.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+<!-- markdownlint-enable line-length -->
 
-[![pipeline status](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/pipeline.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+Simple, safe and secure config management for NodeJS. Based on [SOPS](https://github.com/mozilla/sops).
 
-[![coverage report](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/coverage.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+Have you ever struggled with different config values across local, CI,
+and various cluster environments? If no, did you ever forget to update the
+cluster config after updating the local config? Have you ever been worried about
+the secrecy of your DB credentials and API keys taking the `.env` file approach?
+Ever leaked confidential data by pushing secrets in your `.env` file to the git remote?
+
+Do you agree that just `dotenv` is not enough? **`strong-config` is here to help!**
+
+- Structure configs hierarchically by writing JSON or YAML.
```

Comment:
This list should be prefixed with something e.g. "Strong Config allows you to:"

Reply:
updated